### DISCUSSION TITLE
Background color for not filled bar

### DISCUSSION THREAD

**Discussion by carlosriveroib** [Abhinandan-Kushwaha/react-native-gifted-charts#717] [Q&A] [answered]
It is possible to change the background color for the part of the bar that is not filled? Something like this? Thank you very much

<img width="511" alt="Captura de pantalla 2024-07-19 a las 12 26 22" src="https://github.com/user-attachments/assets/24c2752d-3a35-40fc-8c2c-9cac4703dcc0">

**Comment by Abhinandan-Kushwaha** [comment]
This is not currently supported, but I can suggest you a hacky solution for now.

Use a Stacked bar chart instead of a normal Bar chart (replace `data` with `stackData`).
Each stack will have 2 sections-
1. One with the value of your actual data
2. Second having value = `maxValue-value`

This will render a stacked bar chart where each stack reaches the full height with the base section representing the actual value.
Then you can adjust the colors and border radii of the stack sections to match with your design.

Give it a try and ping here if it doesn't work.

**Reply by carlosriveroib** [reply]
Hi, thanks for the answer. We cannot use a stacked bar chart. But it's ok, that design it is not strictly necessary. I will be reading the changelog. If you finally add this functionality, just write here if you remember. Thank you very much

**Comment by Abhinandan-Kushwaha** [comment, marked as answer]
Hi @carlosriveroib 
This can be achieved using the `showVerticalLines` prop and setting the `verticalLinesThickness` equal to the bars' thickness.
The default bar thickness is 30, so you can set `verticalLinesThickness={30}`.

Next, you can make the verticalLines rounded using `verticalLinesStrokeLinecap='round'`
The `verticalLinesStrokeLinecap` is available prop from version `1.4.22` onwards.

**Reply by carlosriveroib** [reply]
Nice, thank you. Round lines strokes doesn't fit well if overflowTop is used. I use this because I have a custom tooltip and it is being cut by the top line, then I use overflowTop to fix that.

Without overflowTop prop
<img width="271" alt="Captura de pantalla 2024-07-29 a las 10 20 22" src="https://github.com/user-attachments/assets/26317cd0-2f5f-4e16-bd85-46e0e45c47f3">

With overflowTop prop
<img width="259" alt="Captura de pantalla 2024-07-29 a las 10 23 39" src="https://github.com/user-attachments/assets/ed2c9da6-6b3d-40a5-bbf1-699e25d1d7ec">

Thank you

**Comment by Abhinandan-Kushwaha** [comment]
@carlosriveroib Try setting the height of the vertical lines using the `verticalLinesHeight` prop.

**Reply by carlosriveroib** [reply]
works perfectly, thank you
